pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -98,6 +98,8 @@ var (
 	domainNodeMapInstanceLock = &sync.RWMutex{}
 	// azClusterMap maintains a cache of AZ instance name to the clusterMoref in that zone.
 	azClusterMap = make(map[string]string)
+	// azClustersMap maintains a cache of AZ instance name to the clusterMorefs in that zone.
+	azClustersMap = make(map[string][]string)
 	// azClusterMapInstanceLock guards the azClusterMap instance from concurrent writes.
```

Comment:
can we update the comment here since this lock will also guard azClustersMap now?

Reply:
done.